man/systemctl.xml

```diff
@@ -1120,6 +1120,58 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           </para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><command>service-log-level</command> <replaceable>SERVICE</replaceable> [<replaceable>LEVEL</replaceable>]</term>
+
+          <listitem><para>If no <replaceable>LEVEL</replaceable> argument is given, print the current log
+          level of the given service <replaceable>SERVICE</replaceable>. The service must declare the
+          appropriate dbus-service alias in <varname>Names=</varname>
+          (<literal>dbus-<replaceable>name</replaceable>.service</literal>) and must implement the generic
+          <citerefentry><refentrytitle>org.freedesktop.LogControl1</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+          interface. (<filename>systemctl</filename> will use the generic D-Bus protocol to access the
+          <interfacename>org.freedesktop.LogControl1.LogLevel</interfacename> setting for the D-Bus name
+          <replaceable>name</replaceable>.)</para>
+
+          <para>If the optional argument <replaceable>LEVEL</replaceable> is provided, then the command
+          changes the current log level of the service to <replaceable>LEVEL</replaceable>. The log level
+          should be a typical syslog log level, i.e. a value in the range 0…7 or one of the strings
+          <constant>emerg</constant>, <constant>alert</constant>, <constant>crit</constant>,
+          <constant>err</constant>, <constant>warning</constant>, <constant>notice</constant>,
+          <constant>info</constant>, <constant>debug</constant>; see <citerefentry
+          project='man-pages'><refentrytitle>syslog</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+          for details.</para></listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><command>service-log-target</command> <replaceable>SERVICE</replaceable> [<replaceable>TARGET</replaceable>]</term>
+
+          <listitem><para>If the <replaceable>TARGET</replaceable> argument is given, print the current log
```

Comment:
Missing 'no'.

Reply:
Oops, fixed. (I also made the descriptions for service-log-level and service-log-target use the same sentence structure. The error was partially caused by them being different.)